pkg/lib/cockpit/_internal/transport.js

```diff
@@ -111,7 +111,7 @@ class Transport extends EventEmitter {
 
         if (options["channel-seed"])
             this.#channel_seed = String(options["channel-seed"]);
-        if (options.host)
+        if (typeof options.host === 'string')
             transport_globals.default_host = options.host;
```

Comment:
This doesn't do exactly the same though. -- previously, `options.host` could be empty which would count the same as "absent". I don't know if that matters, though.

Reply:
I don't think we've ever sent an empty string for the host field — it defaults to `localhost` if not set.  It's either absent, `null`, `'localhost'` or some other non-empty string.

Unless you know otherwise, I think this change is correct.

Reply:
No, I have no reason to believe that we actually depend on `host: ''` anywhere, it just caught my attention.